bsip-0086.md

```diff
@@ -0,0 +1,119 @@
+fee and order cancellation fee. It seems reasonable that the asset owners
+share some profits (a part of market fees) to the network to support its
+development.
+
+The cut of market fees can go to committee-account's vesting balances.
+As a supporting measure, committee-account should be exempted from
+white-listing restrictions, so that it is able to claim the vesting balances
+and sell them for core token or other tokens to pay for development later.
+
+# Specifications
+
+## Protocol upgrade
+
+A time will need to be scheduled for applying the change. In this document,
+terms "before the protocol upgrade", "at the protocol upgrade" and "after
+the protocol upgrade" may or may not be used to indicate things happen before
+the scheduled time, at the scheduled time and after the scheduled time.
+
+## New global parameter
+
+Add a new global parameter `market_fee_network_percent` which can be updated
+by the committee only after the protocol upgrade.
+Initial value of that parameter is `0`.
+Valid range of that parameter is `[0, 100%]`.
+
+## When processing market fees
+
+After the protocol upgrade, when splitting a non-zero market fee, firstly
+split `amount * market_fee_network_percent` (round down) to committee-account,
+then process the remainder as before. The amount split to committee-account
+should go to the vesting balance object whose type is `market_fee_sharing`.
+
+## When checking asset authorities
+
+After the protocol upgrade, when checking authorities (E.G. white-lists) of an
+asset on an account, if the account is committee-account, let it pass.
+
+# Discussion
+
+* Other than [the "coin-days as market fees" proposal](
+ https://github.com/bitshares/bsips/issues/191) which applies
+ a discount to market fees to benefit core token holders directly, this BSIP
+ seeks for a mechanism to increase the network's income and benefit core token
+ holders indirectly.
+
+* Adjusting the fee schedule and network percent in the referral program is
+ another option to increase the network's income, and is probably sufficient
+ for funding development.
+
+* In the original design, the purpose of market fees is to reward asset
+ issuers for their work (e.g. gateway providers, or businesses built around
+ PMs), while operation fees (`limit_order_create/cancel` in particular) reward
+ the network for performing its work. The separation of market/operation fees
+ matches the separation of roles. Changing this may damage existing businesses.
+ Thus this BSIP could be a bad idea from this perspective. Also note that the
+ network can profit from market fees by setting them on committee-owned tokens.
+
+* This would be another motivation for businesses building on top of BitShares
+ to get involved in the parameter governance process. The percentage can be
+ `0` if it is more appropriate.
+
+* Asset owners can specify a zero market fee percentage to get around the
+ "tax", while still profiting by setting a higher deposit/withdrawal fee.
```

Comment:
replace "tax" with "newly introduced network market fee for asset owners".

a) blockchain lose legally freedom as entire MIT and free concept tech the moment when use the word tax.
b) to be able to "tax" blockchain must have an owner (business registered entity)

Solution: Try to replace tax with something as `network default market fee`

Reply:
Updated to "get around fee sharing".